### ISSUE TITLE
Could not access rapidoc or swagger ...

### ISSUE THREAD

**Issue by santhosh34** [permitio/cedar-agent#6]
Cant load the swagger or rapidoc after running  using docker as per the README instructions. 
docker run permitio/cedar-agent
OS: Mac M1 Pro Ventura

**Comment by asafc** [comment]
Hi @santhosh34 :)

Is this the exact command you ran?
```
docker run permitio/cedar-agent
```

It looks like there are a couple issues here.

First the port needs to be exposed from the docker container, this should be the command:
```
docker run -it -p 8180:8180 permitio/cedar-agent
```

But it looks like there is another issue here. We'll investigate and get back to you.
cc @omer9564

**Comment by omer9564** [comment]
This is fixed in the new Dockerfile , I will release a new docker image and close it afterwards

**Comment by omer9564** [comment]
Fixed in version 0.1.2
